.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimeAltExtractor.cs

```diff
@@ -383,6 +383,11 @@ private void ResolveImplicitRelativeDatePeriod(List<ExtractResult> ers, string t
                 relativeTermsMatches.AddRange(regex.Matches(text).Cast<Match>());
             }
 
+            if (relativeTermsMatches.Count == 0 || (relativeTermsMatches.Count == 1 && config.ThisPrefixRegex.IsMatch(relativeTermsMatches[0].Value)))
```

Comment:
Can you add a code comment here to describe what this 'if' is doing? It will greatly help people better understand the code.